contracts/core/native_fallback/L2/NativeProver.sol

```diff
@@ -55,26 +55,11 @@ contract NativeProver is Ownable, INativeProver {
         uint256 indexed _destinationChainID, uint256 indexed _blockNumber, bytes32 _L2WorldStateRoot
     );
 
-    /**
-     * @notice Block number is too recent to prove
-     */
-    error NeedLaterBlock(uint256 _inputBlockNumber, uint256 _nextProvableBlockNumber);
```

Comment:
Removal of this makes sense. This was previously used by in the async approach (to indicate that we were trying to use a block older than the latest cached block+SETTLEMENT_BLOCKS_DELAY)